scala-package/infer/src/main/scala/ml/dmlc/mxnet/infer/ImageClassifier.scala

```diff
@@ -44,12 +45,19 @@ class ImageClassifier(modelPathPrefix: String,
 
   val classifier: Classifier = getClassifier(modelPathPrefix, inputDescriptors)
 
+  require(inputDescriptors.head.shape.length != 0,
```

Comment:
.length >= 3

Reply:
this check already comes from DataDesc. So, wasnt adding it here. But yeah I can add extra check